airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,6 +24,45 @@
 Changelog
 ---------
 
+5.1.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+
+Features
+~~~~~~~~
+
+* ``Additional mask aws credentials (#26014)``
+* ``Add RedshiftDeleteClusterSnapshotOperator (#25975)``
+* ``Add redshift create cluster snapshot operator (#25857)``
+* ``Add common-sql lower bound for common-sql (#25789)``
+* ``Allow AWS Secrets Backends use AWS Connection capabilities (#25628)``
+* ``Implement 'EmrEksCreateClusterOperator' (#25816)``
+* ``Improve error handling/messaging around bucket exist check (#25805)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Fix display aws connection info (#26025)``
+* ``Fix 'EcsBaseOperator' and 'EcsBaseSensor' arguments (#25989)``
+* ``Fix EMR serverless system test (#25969)``
```

Comment:
I think system test fixes should be skipped from change log.. no real value to users going over these entries. Its internal.

Reply:
Oh yes agreed. The others are

Reply:
Right. I usually put them in "ignored" section.